src/service/action-impl.js

```diff
@@ -492,7 +496,8 @@ export class ActionService {
     let n = target;
     while (n) {
       const actionInfos = this.matchActionInfos_(n, actionEventType);
-      if (actionInfos) {
```

Comment:
@jridgewell We can. But does it help?

It seems that everybody is a bit scared of `el.matches()`. Is there a reason it should not be reliable and fast? And this specific selector is a simple one, just a pseudo-class the browser has already computed itself (for CSS and JS behaviour).

Also, do we really save much (if anything) if we first test the element is a form control?